tests/modes.spec.ts

```diff
@@ -1,19 +1,80 @@
-import { readFile } from "fs/promises";
 import { test, expect } from "@playwright/test";
 
-test("clearing all while a feature is open works", async ({ page }) => {
-  await page.goto("/scheme.html?authority=North Somerset");
-  await page.getByRole("button", { name: "New point" }).click();
+async function clickMap(page, x, y) {
   await page.getByRole("region", { name: "Map" }).click({
-    position: {
-      x: 500,
-      y: 500,
-    },
+    position: { x, y },
   });
+}
+
+test("clearing all while a feature is open works", async ({ page }) => {
+  await page.goto("/scheme.html?authority=Adur");
```

Comment:
Switching to an authority with one of the smallest route files to load, in the hopes that it helps test speed